cmd/agent/flags.go

```diff
@@ -67,6 +67,12 @@ var flags = []cli.Flag{
 		Name:    "hostname",
 		Usage:   "agent hostname",
 	},
+	&cli.StringFlag{
+		EnvVars: []string{"WOODPECKER_AGENT_ID_CONFIG_PATH"},
+		Name:    "agent-id-config-path",
+		Usage:   "agent-id config file path",
+		Value:   "/etc/agent-id.conf",
```

Comment:
```suggestion
		Value:   "/etc/woodpecker/agent-id.conf",
```
?

Reply:
also we might move this value outside into a global defined var ... so we can change it on build time (differ it for docker builds and binarys etc...)

Reply:
> also we might move this value outside into a global defined var ... so we can change it on build time (differ it for docker builds and binarys etc...)

I've been thinking about this a bit, and I'm not sure if it doesn't cause more problems than it ends up simplifying. If the default value is changed at build time, it no longer matches the documentation, nor does it match the default value in the flags.go file. This could confuse users. But at the end of the day, I guess it doesn't make a difference where the agent-id conf file is in the Docker container.

So my suggestion would be to use the path you suggested for all applications and not introduce a globally defined var. What do you think?

Reply:
well agents did not had a state before ... so we should just go with  `/etc/woodpecker/`* ...